#request
string previousString

---
#response
string nextString
